packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -242,7 +242,9 @@ export class PendingStateManager implements IDisposable {
 	 * the batch information was preserved for batch messages.
 	 * @param message - The message that got ack'd and needs to be processed.
 	 */
-	public processPendingLocalMessage(message: SequencedContainerRuntimeMessage): unknown {
+	public processPendingLocalMessage(
+		message: ISequencedDocumentMessage & TypedContainerRuntimeMessage<string, unknown>,
```

Comment:
Do we want an alias `AnyContainerRuntimeMessage` for this?

Reply:
There is an effective Any* used with the new helper `buildPendingMessageContent`. Uses of such things should be limited.